Gemfile

```diff
@@ -42,6 +42,8 @@ gem "twitter-bootstrap-rails"
 # This gem provides jQuery and the jQuery-ujs driver for your Rails 4+ application.
 gem 'jquery-rails', '~> 4.3', '>= 4.3.1'
 
+# This gem contains every available AWS service gem support. 
+gem 'aws-sdk', '~> 3.0', '>= 3.0.1'
```

Comment:
Just have a look if you can achieve the same end result with the `aws-sdk-s3` gem, so you don't load all the other gems. It helps keep your app a bit smaller.